### DISCUSSION TITLE
Checkout@v2 error: Actions in this workflow must within a repository owned by ORGANIZATION

### DISCUSSION THREAD

**Discussion by XiaoleiShi-NV** [community/community#25488] [Actions]
<p>I am trying to setup CI test for our repo (<a href="https://github.com/NVIDIA/HugeCTR" rel="noopener nofollow ugc">https://github.com/NVIDIA/HugeCTR</a>), but when I use the <code>actions/checkout@v2</code>, the following error occurs even without running any job, can anyone provide me some hints about how to solving it?</p>
<p>Error</p>
<p><code>actions/checkout@v2 and actions/checkout@v2 are not allowed to be used in NVIDIA/HugeCTR. Actions in this workflow must be: within a repository owned by NVIDIA.</code></p>
<p>(<a href="https://github.com/NVIDIA/HugeCTR/actions/runs/292598469" rel="noopener nofollow ugc">https://github.com/NVIDIA/HugeCTR/actions/runs/292598469</a>)</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/xiaoleishi-nv">@XiaoleiShi-NV</a>,</p>
<p>Looks like, the owners or administrators of your organization “<strong>NVIDIA</strong>” have set the policy to be “<strong>Allow local actions only</strong>”. This option only allows the actions defined in the repositories within your organization “<strong>NVIDIA</strong>” can be used.<br>
If you want to use the actions from other organizations, you can ask the owners or administrators to select the “<strong>Allow all actions</strong>” option.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/9/194b0ab6701f40f2ea0ae357fb8012fd43a41266.png" data-download-href="/uploads/short-url/3BKIdrFfmze1OebZw30bcCH4rf8.png?dl=1" title="org-settings" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/46307996/181091367-6117d299-e68d-4d79-9b26-798a3ba5227e.png" alt="org-settings" data-base62-sha1="3BKIdrFfmze1OebZw30bcCH4rf8" width="555" height="499" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/9/194b0ab6701f40f2ea0ae357fb8012fd43a41266_2_555x499.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/9/194b0ab6701f40f2ea0ae357fb8012fd43a41266_2_832x748.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/9/194b0ab6701f40f2ea0ae357fb8012fd43a41266.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/9/194b0ab6701f40f2ea0ae357fb8012fd43a41266_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">org-settings</span><span class="informations">1015×914 59.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Another workaround is that you can fork or import the repository of checkout action into your organization “<strong>NVIDIA</strong>”, then use this action within your organization.</p>

**Reply by arbaleast** [reply]
> [@XiaoleiShi-NV](/xiaoleishi-nv),
> 
> Looks like, the owners or administrators of your organization “**NVIDIA**” have set the policy to be “**Allow local actions only**”. This option only allows the actions defined in the repositories within your organization “**NVIDIA**” can be used. If you want to use the actions from other organizations, you can ask the owners or administrators to select the “**Allow all actions**” option.
> [<img alt="org-settings" width="555" height="499" src="https://user-images.githubusercontent.com/46307996/181091367-6117d299-e68d-4d79-9b26-798a3ba5227e.png"> org-settings1015×914 59.4 KB ](https://aws1.discourse-cdn.com/github/original/3X/1/9/194b0ab6701f40f2ea0ae357fb8012fd43a41266.png)
> 
> Another workaround is that you can fork or import the repository of checkout action into your organization “**NVIDIA**”, then use this action within your organization.

Thank you for you help

**Comment by XiaoleiShi-NV** [comment]
<p>Thank you!<br>
This is the correct answer to my question.</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/xiaoleishi-nv">@XiaoleiShi-NV</a>,</p>
<p>Thanks for your rely, and glad that my suggestion can help you solve the problem.<br>
Maybe, you can mark my suggestion as the solution of this topic, so that other people who have the similar questions can notice this when they are looking for an answer.</p>
